.github/workflows/irap-fedora-ci-runner.yaml

```diff
@@ -0,0 +1,49 @@
+name: Build and Push atlas-irap-env container
+
+on:
+  push:
+    branches: 
+      - master
+    paths:
+      - "irap-fedora/**"
+  pull_request:
+    paths:
+      - "irap-fedora/**"
+
+jobs:          
+  deploy:
+    name: Build and deploy image to quay.io/ebigxa
+    runs-on: ubuntu-20.04
+    env:
+      IMAGE_NAME: irap_fedora
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Read tag
+      id: gettag
+      run: echo "::set-output name=tag::$(head -n 1 irap-fedora/tag | awk -F':' '{print $2}')"
+
+    - name: Build image
+      id: build-image
+      uses: redhat-actions/buildah-build@v2
+      with:
+        image: ${{ env.IMAGE_NAME }}
+        tags: ${{ steps.gettag.outputs.tag }}
+        containerfiles: |
+          ./irap-fedora/Dockerfile
+    
+    - name: Push to Quay
+      if: ${{ github.ref == 'refs/heads/master' }}
+      id: push-to-quay
+      uses: redhat-actions/push-to-registry@v2
+      with:
+        image: ${{ steps.build-image.outputs.image }}
```

Comment:
if the quay.io/ebigxa/irap_fedora doesn't exist yet, you might need to create it there before this actually works (see general README on this repo).

Reply:
Done, with appropriate write access.